python/ribasim/tests/test_node.py

```diff
@@ -0,0 +1,40 @@
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+from matplotlib import axes
+from ribasim.node import Node
+
+
+def test(tmp_path):
```

Comment:
It would be good to have a descriptive name for the function.
The only requirement is that its name starts with `test`